mlflow/models/evaluation/default_evaluator.py

```diff
@@ -419,6 +420,44 @@ def _do_plot(**kwargs):
 }
 
 
+def _extract_output_and_other_columns(model_predictions, output_column_name):
```

Comment:
in this function `model_predictions` comes from `self.model.predict`, which can return pandas.DataFrame, pandas.Series, numpy.ndarray or list ([docs](https://mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#mlflow.pyfunc.PyFuncModel.predict)). can we make sure to handle all of these cases?